database/factories/BanFactory.php

```diff
@@ -2,26 +2,26 @@
 
 $factory->define(\App\Models\Ban::class, function (Faker\Generator $faker) {
     return [
-        'user_id' => function (){
+        'user_id' => function () {
```

Comment:
Small thing but this can now be `'user_id' => factory(\App\User::class)`

Reply:
as in without the create? Do you have a link to the docs for this, can't see on the laravel page

Reply:
https://www.neontsunami.com/posts/building-relations-with-laravel-model-factories

We use it in core too if you take a look in waiting lists. But  yeah, reduces the need for the callback and the ->id method call